monitoring/src/check-page/aus.ts

```diff
@@ -23,7 +23,9 @@ const interactWithCMP = async (config: Config, page: Page) => {
 		return;
 	}
 
-	await frame.click('button[title="Continue"]');
+	await frame.click(
+		'div.message-component.message-row > button.sp_choice_type_11',
```

Comment:
This seems like it might be quite brittle if SP change their implementation and it makes it hard to understand what this line of code is doing.

Are we making these changes to allow for the AB tests changing the language? If so, this makes more sense, but I think it would still be good to move these `frame.click`s into functions with more descriptive names or add comments with detail about what the click is supposed to be doing

Reply:
In essence yes - this change would bring these tests in line with the latest [commercial canaries update](https://github.com/guardian/commercial-canaries/pull/35) (on the back of the AB test) which now use the CSS handle, which as far as I can tell is something we define in the SP interface. 
I agree that adding a comment stating what the click is doing would be helpful though.

Reply:
I've altered the method names to make them clearer.